website/docs/tutorials/07-testing/02-cheatcodes/declare-cairo0.md

```diff
@@ -0,0 +1,11 @@
+# `declare_cairo0`
```

Comment:
I don't think it should be included here, we should create corresponding section in cairo1 support section. Cheatcode should be documented in rust. 

Contract based test runner has its own declare and does not support anything cairo1 related.